Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -45,6 +45,8 @@ namespace Multiplayer
 {
     using namespace AzNetworking;
 
+    AZ_CVAR(bool, editor_clientserver, false, nullptr, AZ::ConsoleFunctorFlags::DontReplicate,
```

Comment:
Noticed this in the docs change but should this be editorsv_clientserver for consistency?

Reply:
Done.